dbflute-runtime/src/main/java/org/dbflute/cbean/coption/LikeSearchOption.java

```diff
@@ -55,6 +56,7 @@
     protected List<String> _originalWildCardList;
     protected List<SpecifiedColumn> _compoundColumnList;
     protected List<Integer> _compoundColumnSizeList;
+    protected boolean _isNullcompoundedAsEmpty;
```

Comment:
_isNullcompoundedAsEmpty to _isNull "C" ompoundedAsEmpty

Reply:
> _isNullcompoundedAsEmpty to _isNull "C" ompoundedAsEmpty

sorry, I made a mistake.

Reply:
modified to "_nullCompoundedAsEmpty"

https://github.com/dbflute/dbflute-core/commit/af6b6ecf2249a76b9e15475f082f5f691c8024e0